README.md

```diff
@@ -39,72 +57,16 @@ A specification folder might look like the following:
   |     | - main.c
   |     | - Makefile
   | - test-04.03.sh     -> some script making use of the test-04.03 application
-  | - ....
+  | - ...
 /05-some-other-spec
   | - spec-05.md
   ...
 /README.md
 ```
 
-### Specs document
-
-The release specifications are descriptions of test-cases and certain other
-criteria that have to be tested/applied before each RIOT release. Each
-specification contains a description of it's goal as well as a number of tasks
-that have to be carried out (as much automated as possible).
-
-Each tasks is to be written in the style of a test-case. This means, that the
-following information must be given for each task:
-- pre-requisites (e.g. RIOT application used, hardware used, network topology, )
-- descriptions of how to exactly carry out the task (step-by-step if possible)
-- a precise list of passing/failing criteria, so that there is no doubt if a
-  task was successful or not
-
-### Test scripts and applications
-
-Each release specification should be accompanied by supporting scripts (and RIOT
-applications if needed), that enable to execute it's tasks to the highest degree
-of automation possible. The long-term goal is the complete automated execution
-of all tasks, though there might be some cases where this is not achievable...
```

Comment:
Dito

Reply:
Again, I thought this was a bit obvious... of course we want to use scripts and automate tests...